react/features/whiteboard/functions.ts

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable import/order */
+import md5 from 'js-md5';
+
+// @ts-ignore
+import { getPinnedParticipant } from '../../features/base/participants';
+import { appendURLParam } from '../base/util/uri';
+
+// @ts-ignore
+import { getCurrentRoomId, isInBreakoutRoom } from '../breakout-rooms/functions';
+
+const getWhiteboardState = (state: any): any => state['features/whiteboard'];
+
+/**
+ * Indicates whether the whiteboard is open.
+ *
+ * @param {Object} state - The state from the Redux store.
+ * @returns {boolean}
+ */
+export const isWhiteboardOpen = (state: any): boolean => getWhiteboardState(state).enabled;
+
+/**
+ * Returns the whiteboard id.
+ *
+ * @param {Object} state - The state from the Redux store.
+ * @returns {string|null}
+ */
+export const getWhiteboardId = (state: any): string | null => getWhiteboardState(state).id;
+
+/**
+ * Returns the id of the participant that enabled the whiteboard.
+ *
+ * @param {Object} state - The state from the Redux store.
+ * @returns {string|null}
+ */
+export const getWhiteboardParticipantId = (state: any): string | null => getWhiteboardState(state).participantId;
```

Comment:
Why do we care about who enabled it first?

Reply:
Or is the comment wrong and this really means the ID we gave to the whiteboard participant itself? If it's the latter, I don't think we envision having multiple whiteboards, so why don't we use a single ID?